api/receiver/api.go

```diff
@@ -145,6 +145,7 @@ func (a apiImpl) UpdatePendingTestRun(newRun shared.PendingTestRun) error {
 		if newRun.OSVersion != "" {
 			run.OSVersion = newRun.OSVersion
 		}
+		// nolint:staticcheck // TODO: Fix staticcheck lint error.
```

Comment:
what was the error?

Reply:
This is the [relevant error](https://github.com/web-platform-tests/wpt.fyi/pull/3324/files/4a3a725f7b047c92628f9eb96cc1eb72cb52726c#r1199003356).